### PR TITLE
move tracetools repository

### PR DESCRIPTION
**Pull request by iluetkeb**
We've moved the organizations root URL. Otherwise things are unchanged.

### CODE REVIEW

File: indigo/distribution.yaml
Comment (tfoote) [inline indigo/distribution.yaml]: Indigo is EOL we don't accept changes to the released content in the distro anymore. It's standing as a snapshot of the state. 

You can update the URL for a moved repository in case someone is trying to build from source.

**Review by tfoote — CHANGES_REQUESTED** [review]

File: indigo/distribution.yaml
Comment (iluetkeb) [reply indigo/distribution.yaml]: I tried changing the URL first (see https://github.com/ros/rosdistro/pull/22980/commits/a4e78b6bfb702df3d50cd4b811b3941626cac47e), but since indigo is EOL, that didn't pass the check either...

Both repos are on Github, so we might just leave the old URL in and rely on Github to do the forward, but that's a bit brittle...

Comment (tfoote) [reply indigo/distribution.yaml]: The CI script doesn't have this exception. But we can manually override it with just the URL change if that's the only error and it's just the url update. I think a4e78b6 + e84dcee12cf360d95d9d1e795c913f1354078c6f should be good.

Comment (iluetkeb) [reply indigo/distribution.yaml]: Alright, I've reverted eceec4d94c64210339e206e346c32232a4bfc156

**Comment by clalancette** [comment]
CI is still unhappy because we are touching an EOL distribution.  At this point, I'd suggest just not changing indigo at all, and let the GitHub redirects do their stuff.  Then we can move forward here.

**Review by clalancette — APPROVED** [review]
This looks OK, but we can't merge because Kinetic is currently in a [hold](https://discourse.ros.org/t/preparing-for-kinetic-sync-2019-12-11/11907).  We generally recommend one PR per ROS distribution to avoid this problem, but here we can just wait until next week and then merge this.

**Review by tfoote — APPROVED** [review]

**Pull request merged**
